.github/workflows/docker-publish.yml

```
@@ -47,6 +47,11 @@ jobs:
         with:
           cosign-release: 'v2.2.4'
 
+      # Set up QEMU so Buildx can build arm64 images on the hosted runner
+      # https://github.com/docker/setup-qemu-action
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@68827325e0b33c7199eb31dd4e31fbe9023e06e8 # v3.0.0
```

YAMLlint (1.38.0)

[warning] too few spaces before comment: expected 2

(comments)